frameworks/hdfs/build.gradle

```diff
@@ -0,0 +1,49 @@
+group 'com.mesosphere.sdk.hdfs'
+version '1.0-SNAPSHOT'
+
+apply plugin: 'java'
+apply plugin: 'application'
+
+repositories {
+    maven {
+        url "http://downloads.mesosphere.com/maven/"
+    }
+    maven {
+        url "http://downloads.mesosphere.com/maven-snapshot/"
+    }
+}
+
+ext {
+    mesosVer = "1.0.1"
+    junitVer = "4.12"
+    systemRulesVer = "1.16.0"
+    mockitoVer = "1.9.5"
+}
+
+dependencies {
+    compile project(":scheduler")
+    compile project(":executor")
+    compile group: 'org.slf4j', name: 'slf4j-simple', version: '1.7.21'
+    compile "org.apache.mesos:mesos:${mesosVer}"
```

Comment:
I have set it at the very root layer, so everything by default is in sync.  Subprojects may override as desired.